account.proto

```diff
@@ -33,7 +33,11 @@ message GenesisPayload {
 
 message Account {
     uint64 balance = 1;
-    repeated string pub_keys = 2;
+    uint64 pub_key_nonce = 2;
```

Comment:
Maybe `pub_key_serial_number`? This would be more descriptive, nonce is usually considered as something random.

Reply:
Ethereum use nonce as incremental. But if it strictly needed a var as number, I will change. Maybe, in future we could use a random nonce for operating. Nonce is more unique here.

Reply:
Ethereum use nonce as incremental. But if it strictly needed a var as number, I will change. Maybe, in future we could use a random nonce for operating. Nonce is more unique here.